Python/Distance-Calculator-App/README.md

```diff
@@ -0,0 +1,28 @@
+# Distance Calculating App
+This app calculates distance between two geo-locations.
+
+# Quick Start:
+- Clone this repository.
+
+      git clone https://github.com/HarshCasper/Rotten-Scripts.git
+      
+- Change Directory
+
+      cd .\Rotten-Scripts\
+      cd Python
+      cd .\Distance-Calculator-App\
+      
+- Install requirememnts
+
+      pip install requirements.txt
+      
+- Run python file
+
+      python main.py
+      
+# Screenshot
+
+![](screenshot/cap.png)
```

Comment:
Remove the screenshot from the commit. Upload the Image on Imgur and reference it here.